share/spice/public_holidays/record.handlebars

```diff
@@ -3,8 +3,8 @@
         {{#each holidays}}
             <tr class="record__row  {{#if ../meta.options.rowHighlight}}record__row--highlight{{/if}}">
                 <td class="record__cell  record__cell--key">{{date}}</td>                
-                <td class="record__cell  record__cell--key">{{name}}</td>                
-                <td class="record__cell  record__cell--key">{{notes}}</td>                
+                <td class="record__cell  record__cell--value">{{name}}</td>
+                <td class="record__cell  record__cell--value">{{notes}}</td>
```

Comment:
@sekhavati to make the most use of space, we should only put in this cell if `notes` is defined. Otherwise we have a blank 3rd column for the majority of results:

```
{{#if notes}}
<td class="record__cell  record__cell--value">{{notes}}</td>
{{/if}}
```